src/main/java/fr/socrates/api/DTO/ConfirmationDTO.java

```diff
@@ -17,10 +20,10 @@
     public ConfirmationDTO() {
     }
 
-    public ConfirmationDTO(String email, AccommodationChoice accommodationChoice, Payment payment) {
+    public ConfirmationDTO(String email) {
```

Comment:
What ?  why single room and payment by transfer by default ?

Reply:
domainToDto is the only caller for that constructor. It is provided with a candidate... It's unfortunate, but we'll have to make a lot of changes to fix that :(